### PR TITLE
Add apps/x509 -set_issuer option

### PR DESCRIPTION
**Pull request by job**
This changeset adds the counterpart to the '-subj' option to allow overriding the Issuer.

The issuer can be specified as following `apps/openssl x509 -new -set_issuer '/CN=example-nro-ta' -subj '/CN=2a7dd1d787d793e4c8af56e197d4eed92af6ba13' ...`

This is useful in constructing specific test-cases or rechaining PKI trees

Joint work with @geeohgeegeeoh

##### Checklist
- [x] documentation is added or updated
- [x] tests are added or updated

### CODE REVIEW

File: apps/x509.c [hunk 1]
```diff
@@ -138,6 +138,7 @@ const OPTIONS x509_options[] = {
      "Number of days until newly generated certificate expires - default 30"},
     {"preserve_dates", OPT_PRESERVE_DATES, '-',
      "Preserve existing validity dates"},
+    {"issu", OPT_ISSU, 's', "Set or override certificate issuer"},
```
Comment (t8m) [inline apps/x509.c:141]: I'd propose to rename the option to `set_issuer`

**Review by t8m — CHANGES_REQUESTED** [review]
We will need a CLA to be signed to accept this contribution from both authors.

Could you please sign it? https://www.openssl.org/policies/cla.html

**Comment by job** [comment]
> We will need a CLA to be signed to accept this contribution from both authors.
> 
> Could you please sign it? https://www.openssl.org/policies/cla.html

Yup, I sent it a minute ago to legal@

File: apps/x509.c (hunk 1 above)
Comment (job) [reply apps/x509.c:141]: Should we alias `-set_subject` to `-subj` for consistency?

Comment (t8m) [reply apps/x509.c:141]: Yes, that would make sense.

**Comment by geeohgeegeeoh** [comment]
Sorry for delay, I've just sent in my ICLA.

cheers

-George

**Pull request closed without merging**